examples/08-spot-instances.yaml

```diff
@@ -0,0 +1,22 @@
+# A simple example of ClusterConfig object:
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+    name: martina-test-spotinst-mixed-9
+    region: eu-central-1
+
+nodeGroups:
+    - name: ng-6
+      ssh:
+          publicKeyPath: ~/.ssh/id_rsa_tests.pub
+      minSize: 2
+      maxSize: 5
+      instancesDistribution:
+        maxPrice: 0.017
+        instanceTypes: ["t3.small", "t3.medium"] # Required because we need more instance types
```

Comment:
Should we say something more specific, e.g. "at least to instance types are required"?